journal-week-1.md

```diff
@@ -0,0 +1,20 @@
+# One Down Looking Forward for Eleven More 
+
+
+The first week was a shock, I indeed entered a boot camp. Starting from the prep-work all throughout the rest of the week, all of us from the class had the taste of brain squeezing experience. I will have to admit that I had second thought of continuing this course go to my back-up plan.  Good thing, by Thursday or Friday, things that we were trying to learn made sense as a whole. We create things in Git, make a copy in our local computer, play with it and test it, bring back to Git.
+
+In particular, these are the things we learned in a short span of one week:
+
+* Github - we created repos in this wonderful version control system. From here we can share, fork our working files. We also learned how to sync my files to our local computer to work with the files and then push it back to Github again,
+
+* Javascript - we learned the foundation this language, starting from the basic types of objects and manipulating these to form simple apps.
+
+* Creating and Testing JS Files - we learned to test our JS files, using some codes in the terminal such as `node`, `mocha` and `chai`. These test frameworks had given me different perspective in testing my js apps other than the conventional console.
+ 
+* Talk & Think like a developer - the first week gave me gave me the idea on what my life as Developer will be. Staring at codes the whole day. haha. Well I honestly enjoyed coding and I get excited when my expected outputs are starting to show up.
```

Comment:
would like to see you grab one of these topics and expand/reflect on them or the learning process.